### DISCUSSION TITLE
🧾 Bill Pay demo

### DISCUSSION THREAD

**Discussion by mcclowes** [codatio/codat-docs#324] [Show and tell]
![pika-1681901649489-1x](https://user-images.githubusercontent.com/4573407/233053686-438d4ef8-2a92-408e-92b3-f35dd1eb7037.png)

This demo shows how you could build a bill pay solution with Codat. It demonstrates how we can manage bill payments of apps like Quickbooks Sandbox.

[See it in action](https://demo-bill-pay.vercel.app/) or [dig into the code](https://github.com/codatio/demo-bill-pay).

**Comment by dcoplowe** [comment]
We've released our bill pay guide! To learn more about how to build your own bill payment solution check out the guide [here](https://docs.codat.io/accounting-api/guides/bill-pay/introduction).
